account_financial_risk/models/res_partner.py

```diff
@@ -567,3 +567,79 @@ def open_risk_pivot_info(self):
             "context": self.env.context,
             "domain": domain,
         }
+
+    def _get_financial_risk_lines(self):
+        # Returns [(flag, value, label), ...] already evaluated for the partner itself.
+        self.ensure_one()
+        return [
+            (
+                self.risk_invoice_draft_include,
+                self.risk_invoice_draft,
+                self._fields["risk_invoice_draft"].string,
+            ),
+            (
+                self.risk_invoice_open_include,
+                self.risk_invoice_open,
+                self._fields["risk_invoice_open"].string,
+            ),
+            (
+                self.risk_invoice_unpaid_include,
+                self.risk_invoice_unpaid,
+                self._fields["risk_invoice_unpaid"].string,
+            ),
+            (
+                self.risk_account_amount_include,
+                self.risk_account_amount,
+                self._fields["risk_account_amount"].string,
+            ),
+            (
+                self.risk_account_amount_unpaid_include,
+                self.risk_account_amount_unpaid,
+                self._fields["risk_account_amount_unpaid"].string,
+            ),
+        ]
+        # return [
+        #     (
+        #         self.risk_invoice_draft_include,
+        #         formatLang(
+        #             self.env,
+        #             self.risk_invoice_draft,
+        #             currency_obj=self.risk_currency_id,
+        #         ),
+        #         self._fields["risk_invoice_draft"].string,
+        #     ),
+        #     (
+        #         self.risk_invoice_open_include,
+        #         formatLang(
+        #             self.env, self.risk_invoice_open, currency_obj=self.risk_currency_id
+        #         ),
+        #         self._fields["risk_invoice_open"].string,
```

Comment:
There are some left over comments, is it intended?

Reply:
Residual code removed, thanks